SU-HSDO/suhumsci
# SHS-43: Contrast failures on pill on certain color palettes

# READY FOR REVIEW

## Summary
Changed the global value of the `hb-pill` to fix contrast errors.

![image](https://user-images.githubusercontent.com/72449433/170362431-c246e7fb-5300-4f66-b109-4dcb33f646ef.png)


## Need Review By (Date)
Whenever anyone is free

## Urgency
This ticket was in the backlog

## Steps to Test
1.  Go [here](http://sparkbox-sandbox.suhumsci.loc/courses/honors-thesis-research/12) run a tool to test for contrast. I personally used the WAVE evaluation tool (it is a browser addon)
2. Ensure that these two pills, do not have contrast errors:

![image](https://user-images.githubusercontent.com/72449433/170362846-181ec991-9777-4d47-9591-0e53438114a7.png)


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
